code/modules/backgrounds/location/_location_events.dm

```diff
@@ -4,27 +4,27 @@
 /decl/location_event/big_game_hunters/announce(var/decl/background_detail/location)
 	. = "Game hunters on [location.name] "
 	if(prob(33))
-		. += "were surprised when an unusual species experts have since identified as \
+		. += "were surprised when an unusual species that experts have since identified as \
```

Comment:
this was technically grammatically correct already. for more information: it's a *defining relative clause,* which doesn't need a relative pronoun and is not marked by commas before and after. it's not *in*correct to include 'which' or 'that', but it's not strictly necessary either.

Reply:
Maybe not strictly necessary, but w/out `that` the sentence comes a little close to being a garden path.
> Game hunters on PLACE were surprised when an (unusual species experts have since identified as THING) turned up.

I think w/out the tailing `turned up` this would most likely read just as well to me.
My usual approach is to read smth like this out load and if it sounds funny, it _is_  funny.

Reply:
Maybe not strictly necessary, but w/out `that` the sentence comes a little close to being a garden path.
> Game hunters on PLACE were surprised when an (unusual species experts have since identified as THING) turned up.

I think w/out the tailing `turned up` this would most likely read just as well to me.
My usual approach is to read smth like this out load and if it sounds funny, it _is_  funny.